### PR TITLE
[coverage]  AS resolvers tests

### PR DESCRIPTION
**Pull request by guedou**
These are simple tests to check the behavior of classes implemented in `scapy/as_resolvers.py`

### CODE REVIEW

File: test/regression.uts [hunk 1]
```diff
@@ -641,6 +641,15 @@ DNS in dns_ans
...
 IP(src="8.8.8.8").whois()
 
+= AS resolvers
+~ netaccess IP
+
+ret = conf.AS_resolver.resolve("8.8.8.8", "8.8.4.4")
+len(ret) == 2
+
+import operator
+reduce(operator.and_, [ x[1] == 15169 for x in ret]) == True
```
Comment (p-l-) [inline test/regression.uts]: You can replace the two previous lines with:
```Python
all(x[1] == 15169 for x in ret)
```

File: test/regression.uts [hunk 2]
```diff
@@ -641,6 +641,15 @@ DNS in dns_ans
 ~ netaccess IP
 IP(src="8.8.8.8").whois()
 
+= AS resolvers
+~ netaccess IP
+
+ret = conf.AS_resolver.resolve("8.8.8.8", "8.8.4.4")
+len(ret) == 2
```
Comment (p-l-) [inline test/regression.uts]: You might want to use `assert` here, since this is not the last line of the test.

Comment (guedou) [inline test/regression.uts]: Done. I always forgot this behavior ...

File: test/regression.uts (hunk 1 above)
Comment (guedou) [inline test/regression.uts]: Done. That's way cleaner. I did not know `all()`.

**Pull request merged**